spec.bs

```diff
@@ -2078,8 +2076,11 @@ and a [=real time reporting contributions map=] |realTimeContributionsMap|:
                   |executionMetrics|) be |generateBidResult|.
           1. Let |generateBidDuration| be the [=duration from=] |generateBidStartTime| to |settings|'s
             [=environment settings object/current monotonic time=], in milliseconds.
-          1. If |perBuyerCumulativeTimeout| is not null, decrement |perBuyerCumulativeTimeout| by
+          1. [=Expend cumulative timeout time=] given |cumulativeTimeoutTracker| and
             |generateBidDuration|.
+          1. If |cumulativeTimeoutTracker| [=cumulative timeout tracker/is expired=] then:
+            1. Increment |metrics|'s [=per participant metrics/cumulative timeouts occurred=] by 1.
+            1. [=iteration/continue=].

```

Comment:
Note the change here; this makes it so the thing affected by cumulative timeout doesn't take effect, which is what.

...And also means it doesn't count as a regular timeout, which you asked about before.